README.unittests.rst

```diff
@@ -199,6 +198,13 @@ Additional steps specific to individual databases are as follows::
         test=# create extension hstore;
         CREATE EXTENSION
 
+    To include tests for CITEXT, create the CITEXT extension::
+
+        postgres=# \c test;
+        You are now connected to database "test" as user "postgresql".
+        test=# create extension citext;
```

Comment:
can you add the command below, in the docker instructions?

> `docker exec -ti postgres psql -U scott -c 'CREATE SCHEMA test_schema; CREATE SCHEMA test_schema_2;CREATE EXTENSION hstore;' test`

Reply:
If I understand correctly, this can be even removed, in case we put it into `provision.py`?

Reply:
yes, probably. We could just have that in the docker command. (since there is if not exists in provision it should be fine if the db already has it enabled)